### DISCUSSION TITLE
"Create issue" button missing on Android when issue list accessed via homescreen shortcut

### DISCUSSION THREAD

**Discussion by archon810** [community/community#31440] [Mobile]
On Android, if I go to a repo, then issues, I can create a new issue using the icon at the top right.

But if I get to the issue list using a homescreen shortcut, the Create issue button is not there. Instead, there are edit/delete shortcut buttons.

Please retain the Create issue button there, otherwise it defeats the benefit of shortcuts.

**Comment by DenverCoder1** [comment]
While shortcuts *can* be filtered for a single repository, they don't have to be. They can be showing issues from several repositories, so I don't think it makes much sense for there to be a create issue button there.

I'd suggest maybe making a shortcut to the repository by adding it to favorites, that way you can get to the issues with only a couple taps from the homescreen.

**Reply by archon810** [reply]
I see what you're saying, I wasn't aware shortcuts could reference multiple repos. However, I can offer an enhancement here.
1. If there's only one repo in a shortcut, the Create issue button would be straightforward.
2. If there is more than one, the Create issue button could offer a list of repos that the shortcut covers (or if it's all repos, then the most recent repos?).

Finally, regarding your suggestion about adding a favorite, I already do that but having a shortcut saves a tap and some time. It would only make sense that at least single repo shortcuts, like web shortcuts to a repo's Issues page, would be an uncompromised experience.
